src/main_nep/parameters.cu

```diff
@@ -172,7 +174,7 @@ void Parameters::calculate_parameters()
     lambda_e = lambda_f = 0.0f;
     enable_zbl = false;
     if (!is_lambda_v_set) {
-      lambda_v = 1.0f;
+      lambda_v = 1.0f; // by default, dipole or polarizability is fitted with global quantities
```

Comment:
Is the same `lambda_v` used for atomic and global virials? You might have to double check the relative weight so that the default value of `lambda_v` still make sense for atomic virials.

Reply:
I think it should be fine, since as far as I can tell `lambda_v` is multiplied by the per-atom virials in both cases already.